Dockerfile

```diff
@@ -156,9 +156,16 @@ RUN echo 'deb http://cz.archive.ubuntu.com/ubuntu kinetic main universe' >> /etc
 # NOTE: This is a temporary solution until changes land upstream.
 ARG TIMESCALE_STATIC_PRIMARY
 RUN if [ "${TIMESCALE_STATIC_PRIMARY}" != "" ]; then \
-    wget -qO- https://raw.githubusercontent.com/timescale/patroni/v2.2.0-beta.4/patroni/ha.py > /usr/lib/python3/dist-packages/patroni/ha.py && \
-    wget -qO- https://raw.githubusercontent.com/timescale/patroni/v2.2.0-beta.4/patroni/config.py > /usr/lib/python3/dist-packages/patroni/config.py && \
-    wget -qO- https://raw.githubusercontent.com/timescale/patroni/v2.2.0-beta.4/patroni/validator.py > /usr/lib/python3/dist-packages/patroni/validator.py; \
+    mkdir /tmp/patroni && cd /tmp/patroni && \
+    git init && git remote add -f origin https://github.com/timescale/patroni.git && \
+    git config core.sparseCheckout true && echo 'patroni' > .git/info/sparse-checkout && \
+    git pull origin feature-static-primaries; \
+    fi
+
+# Update Patroni package dir with new code.
+RUN if [ "${TIMESCALE_STATIC_PRIMARY}" != "" ]; then \
```

Comment:
Any reason why this `RUN` is separated from the one in L158 ?

Reply:
It can be bundled, I was just copying over what was asked from here https://github.com/timescale/forge-issues/issues/605

Reply:
@nhudson yea, I think we should go ahead and collapse this down into a single `RUN` block. I think there is a non-zero risk that the blocks could get out of sync. In that reference issue that I put together, I mostly did it that way for rapid testing/debugging.

Reply:
Eh, disregard. Let's just keep this as-is. We will be swapping it out for the new failsafe mode in Patroni soon.